readme.md

```diff
@@ -1,7 +1,43 @@
+# EarTrainingApp
```

Comment:
badges should be here! look at this example:
https://github.com/facebook/react

at least have 2 badges:  build status & test coverage